service/tcp.go

```diff
@@ -117,24 +118,26 @@ type tcpService struct {
 	running     sync.WaitGroup
 	readTimeout time.Duration
 	// `replayCache` is a pointer to SSServer.replayCache, to share the cache among all ports.
-	replayCache    *ReplayCache
-	checkAllowedIP func(net.IP) *onet.ConnectionError
+	replayCache       *ReplayCache
+	isTargetIPAllowed func(net.IP) *onet.ConnectionError
```

Comment:
There's something odd about a function named `is*` that doesn't return a `bool`.  Maybe `targetIPValidator` would be a good name?

Reply:
Renamed